Formula/w/w3m.rb

```diff
@@ -2,29 +2,23 @@ class W3m < Formula
   desc "Pager/text based browser"
   homepage "https://w3m.sourceforge.net/"
   license "w3m"
-  revision 8
   head "https://github.com/tats/w3m.git", branch: "master"
 
   stable do
-    url "https://deb.debian.org/debian/pool/main/w/w3m/w3m_0.5.3.orig.tar.gz"
-    sha256 "e994d263f2fd2c22febfbe45103526e00145a7674a0fda79c822b97c2770a9e3"
+    url "https://deb.debian.org/debian/pool/main/w/w3m/w3m_0.5.3+git20230121.orig.tar.xz"
+    sha256 "974d1095a47f1976150a792fe9c5a44cc821c02b6bdd714a37a098386250e03a"
+    version "0.5.3_git20230121"
```

Comment:
Not sure on how to best convert the `+`. This is one of the symbols we don't support. Options are:
- `0.5.3_git20230121`
- `0.5.3-git20230121`
- `0.5.3.git20230121`
- `0.5.3git20230121`

Reply:
In order of preference: `-`, `.`, none (`0.5.3git20230121`), and then `_`.

But my reasoning is purely just aesthetic. Open to an alternative ordering based on other reasons.